pipelines/template-build/template-build.yaml

```diff
@@ -226,7 +226,20 @@ spec:
       workspaces:
       - name: registry-auth
         workspace: registry-auth
-
+    - op: add
+      path: /spec/tasks/-
```

Comment:
it's task directly, not jsonpatch format

Reply:
Sorry....., I just copy it, not read it carefully